### PR TITLE
Feature/display issue page details

### PR DESCRIPTION
**Pull request by efren02**
GitHub Link
https://github.com/framgia/sph-els-efren/tree/feature/display_issue_page_details
Asana Link
- 
https://app.asana.com/0/1202711011113639/1202767063767870/f

Commands

` npm i -D react-router-dom`
Definition of done

- Updated Filter And Pagination Code
- When Page Logo is click Redirected to home page. 
-When Labels Button is clicked Redirected to Labels Table.
Created Redirect Link for titles
- Optimize Link & Route using React-Route-Dom
Notes
 The label field in the issue table is still being fixed
Screenshots
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,5 +35,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-router-dom": "^6.3.0"
```
Comment (chevyB) [inline package.json]: it should be dependencies not a devDependencies because it is needed in deployment

File: src/components/Content.js [hunk 2]
```diff
@@ -1,37 +1,67 @@
...
     count = active.length;
     count2 = closed.length;
+    
+    const loadAuthor = (term) => {
+        console.log(term)
+    }
+
+    useEffect(() => {
+
+    }, [])
```
Comment (chevyB) [inline src/components/Content.js]: What are these for?

File: src/components/Content.js [hunk 3]
```diff
@@ -1,37 +1,67 @@
...
+
+    const renderedList = issues.map((issue) => {
+        let labelName = '';
+        
+        issue.labels.map((label) => {
+            labelName = label.name
+         })
```
Comment (chevyB) [inline src/components/Content.js]: So only 1 label is displayed? What about other labels?

File: src/components/Header.js [hunk 4]
```diff
@@ -1,38 +1,38 @@
...
-                        <input type="text" placeholder='Search or jump to...' />
-                        <i className="inverted search link icon"></i>
-                    </div>
+            <div className="header item">
+                <Link to="/"><i className="big github icon inverted" /></Link>
+            </div>
+            <div className='header item'>
+                <div className="ui transparent icon input">
+                    <input type="text" placeholder='Search or jump to...' />
+                    <i className="inverted search link icon"></i>
```
Comment (chevyB) [inline src/components/Header.js]: Single tag

File: src/components/Header.js [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
-                        <div className="menu">
-                            <div className="item">Applications</div>
-                            <div className="item">International Students</div>
-                            <div className="item">Scholarships</div>
-                        </div>
-                    </div>   
-                </div>
+            <div className='header item right aligned'>
+                <div className="ui right dropdown item">
+                    <i className="dropdown icon"></i>
```
Comment (chevyB) [inline src/components/Header.js]: Single tag

File: src/components/Content.js [hunk 6]
```diff
@@ -1,37 +1,67 @@
...
 
-    const renderedList = issues.map((issue, index) => {
         return(
+          
             <tr key={issue.id}>
                 <td className="collapsing">
                     <i className="dot circle outline large green icon"> </i>
-                    {issue.title}
+                    <a href={issue.html_url} style={{ color:'white' }} target="_blank" alt="issues"> {issue.title}</a>
```
Comment (chevyB) [inline src/components/Content.js]: You should create your detail page. Dont redirect it to github detail

**Review by chevyB — CHANGES_REQUESTED** [review]

File: src/components/Content.js (hunk 6 above)
Comment (efren02) [reply src/components/Content.js]: ok sir :)

File: src/components/Content.js (hunk 3 above)
Comment (efren02) [reply src/components/Content.js]: eh loop niya sir per title sir

File: src/components/Content.js (hunk 2 above)
Comment (efren02) [reply src/components/Content.js]: wala pa na tapos sir. wala nako na stash.

**Review by chevyB — COMMENTED** [review]
- [x] The id is placed incorrectly, you should follow what github looks like.
- [x] Header label is in bold, github isn't 
- [x] Date info should be same as github "opened time ago"
- [x] should show who is the person opened the issue
- [x] Update screenshot including pagination

File: src/components/Content.js (hunk 3 above)
Comment (chevyB) [reply src/components/Content.js]: @efren02 it should display all tags, not only one

**Review by chevyB — APPROVED** [review]

**Pull request merged**